exercicio1/countvowels.ts

```diff
@@ -0,0 +1,22 @@
+function getCount(str: string){
+    str = str.toLowerCase();
+    var count = 0;
```

Comment:
Outra boa prática é usar `let` e `const` em vez de `var` na maioria das situações em JavaScript/TypeScript.

o `var` tem escopo de função, o que significa que a variável é acessível em toda a função, mesmo antes de sua declaração. Isso pode causar confusão se a variável for usada antes de ser inicializada.

let e const têm escopo de bloco, ou seja, são acessíveis apenas dentro do bloco onde foram declaradas (como dentro de um loop ou uma condicional). Isso ajuda a evitar problemas de escopo e torna o código mais previsível.